### ISSUE TITLE
Welcome dialog: accessibility for screen reader users.

### ISSUE THREAD

**Issue by DavidBailes** [audacity/audacity#8798]
### Bug description

So that screen reader users can realise that there is a carousel, and how it can be interacted with, as well to be able to read some version of the slide title and image,  the accessibility names of the three carousel buttons need to provide this information.
Proposed names:
The left and right arrow buttons have the accessibility names "previous slide", and "next slide".
The button below the image has an accessibility names such as:
"Slide 1 of 5, Complete your Audacity cloud setup with Audio.com. Continue"
"Slide 2 of 5, What's new in Audacity. Watch the release video"
"Slide 3 of 5, Soap Voice Cleaner: studio-quality voice-over sound. Get it on MuseHub"
"Slide 4 of 5, Help us decide the future of Audacity, Audacity feature survey. Take part in Survey"
"Slide 5 of 5, 25th Anniversary Merchandise!, Visit our new Audacity merch store. Visit now"

### Steps to reproduce

With a screen reader running, tab around the dialog

### Expected behavior

A screen reader used should be able to interact with the "carousel" and be able to read the information which is provided by images.

### Actual behavior

They can't

### Audacity Version

Audacity3

### Operating system

Windows

### Additional context

This issue can only be fixed after this issue is fixed: https://github.com/audacity/audacity/issues/8796

**Comment by DavidBailes** [comment]
This issue can only be fixed after https://github.com/audacity/audacity/issues/8796 is fixed.

**Comment by chinakov** [comment]
- The arrows to enable scrolling of the carousel are read by the screen reader as: Panel-Pane, and have no specificity to which arrow has been selected (Left or Right). 
- When the Left or Right carousel arrow is used via tabbing and 'Enter', it closes the Welcome Dialog. It should change the carousel items.

**Comment by chinakov** [comment]
One more: Improvement to make the screen reader read the title of each carousel item, shown here: 

![Image](https://github.com/user-attachments/assets/59a323ea-834c-4544-9afa-a4417a19206d)

**Comment by chinakov** [comment]
Thank you, @DavidBailes

**Comment by chinakov** [comment]
Tooltips for arrow buttons are implemented.

**Comment by DavidBailes** [comment]
@grliszas14 . I presume that the tooltip on the middle button is only intended to be temporary, and shouldn't be present after the buttons have been given accessibility names.
Is that correct?

**Comment by grliszas14** [comment]
@DavidBailes yes, that is correct

**Comment by DavidBailes** [comment]
> [@DavidBailes](https://github.com/DavidBailes) yes, that is correct

OK, I'll create a PR to set the accessibility names.

**Comment by chinakov** [comment]
Tested:


- [x] The left and right arrow buttons have the accessibility names "previous slide", and "next slide".
- [x] The button below the image has an accessibility names such as:
- [x] "Slide 1 of 5, Complete your Audacity cloud setup with Audio.com. Continue"
- [x] "Slide 2 of 5, What's new in Audacity. Watch the release video"
- [x] "Slide 3 of 5, Soap Voice Cleaner: studio-quality voice-over sound. Get it on MuseHub"
- [x] "Slide 4 of 5, Help us decide the future of Audacity, Audacity feature survey. Take part in Survey"
- [x] "Slide 5 of 5, 25th Anniversary Merchandise!, Visit our new Audacity merch store. Visit now"
